templates/scenarios/command-and-response-js/.fx/appPackage/manifest.template.json

```diff
@@ -142,7 +142,7 @@
         "messageTeamMembers"
     ],
     "validDomains": [
-        "{{state.fx-resource-bot.domain}}",
-        "{{state.fx-resource-bot.domain}}"
+        "${{BOT_DOMAIN}}",
+        "${{BOT_DOMAIN}}"
```

Comment:
We only need one ${{BOT_DOMAIN}}, could you help remove the redundancy?

Reply:
fixed